README.md

```diff
@@ -38,8 +38,8 @@ Takipi installs a daemon process and a JVM agent library on your servers. These
 
 Requires;
 
-* `puppetlabs-stdlib`
-* `puppetlabs-java`
+* `puppetlabs-stdlib`         (puppet module install puppetlabs-stdlib)
```

Comment:
please also make sure to update the version and add your changes to the CHANGELOG in the readme